Makes the udev rules run a script to start brltty instead of just
running it directly.
diff --git a/Autostart/Udev/udev.rules b/Autostart/Udev/udev.rules
index 006d034..eedb7c4 100644
--- a/Autostart/Udev/udev.rules
+++ b/Autostart/Udev/udev.rules
@@ -439,15 +439,14 @@ ENV{BRLTTY_BRAILLE_DEVICE}="usb:vendor=0X$sysfs{idVendor}+product=0X$sysfs{idPro
 GOTO="brltty_run"
 
 LABEL="brltty_run"
-ENV{BRLTTY_PID_FILE}="/run/brltty.$kernel.pid"
-ACTION=="add", GOTO="brltty_add"
+ENV{BRLTTY_PID_FILE}="/var/run/brltty/brltty.$kernel.pid"
+ACTION=="add|change", GOTO="brltty_add"
 ACTION=="remove", GOTO="brltty_remove"
 GOTO="brltty_end"
 
 LABEL="brltty_add"
-SYMLINK+="brltty/$env{BRLTTY_BRAILLE_DEVICE}"
-TEST=="/sys/fs/cgroup/systemd", GOTO="brltty_add_systemd"
-RUN+="/bin/brltty -E
+SYMLINK+="brltty/$env{BRLTTY_BRAILLE_DRIVER}-$env{BRLTTY_BRAILLE_DEVICE}"
+RUN+="/lib/udev/brltty"
 GOTO="brltty_end"
 
 LABEL="brltty_add_systemd"
